README.md

```diff
@@ -31,6 +31,26 @@ It needs to be given the following variable to work:
 | `DISTRO_REVISION`  | None    | Eg, "1.0.3"             |
 | `K8S_DESCRIPTION_FILES_GIT_REF`  | `master`   | The Git revision of the K8s files to be used for deployment. Eg, "7a1e77398560b914ba4a02e19f2b066d55c1347f"             |
 
+## 'restore' prfile
+
+This is a profile that will download and package all binaries and configurations needed to restore databases and fielstore and save it back in the usb device 
```

Comment:
This profile will prepare scripts to restore database and filestores on an existing cluster.